Makefile

```diff
@@ -3,9 +3,15 @@ PREFIX ?= ${HOME}/opt
 DB_DIR = ${PREFIX}/nextpnr/prjxray-db
 CHIPDB_DIR = ${PREFIX}/nextpnr/xilinx-chipdb
 XRAY_DIR ?= ${PREFIX}/prjxray
+NEXTPNR_DIR ?= ${PREFIX}/nextpnr
 SHELL = /bin/bash
 PYTHONPATH ?= ${XRAY_DIR}
 
+ifeq ($(shell uname -s),Darwin)
```

Comment:
Can you add a comment to explain that this extension refers to Mac (Intel?) only, please?

Reply:
# This workaround is only required for macOS, because the default clang does not provide libomp.